Makefile

```diff
@@ -72,7 +72,8 @@ $(GOFUZZBUILD):
 
 GOFUZZ = $(GOBIN)/go-fuzz
 $(GOFUZZ):
-	go get $(GOGETFLAGS) github.com/dvyukov/go-fuzz/go-fuzz
+	go get $(GOGETFLAGS) github.com/mdempsky/go114-fuzz-build
+	ln -s $(GOPATH)/bin/go114-fuzz-build $(GOPATH)/bin/go-fuzz
```

Comment:
This should have changed the GOFUZZBUILD rule above instead of this one, avoiding the symlink and the change of meaning.